ss/ss.go

```diff
@@ -8,76 +8,45 @@ import (
 	"strings"
 	"time"
 
+	log "github.com/sirupsen/logrus"
 	corev1 "k8s.io/api/core/v1"
 
-	"github.com/openshift-kni/commatrix/client"
 	"github.com/openshift-kni/commatrix/consts"
 	"github.com/openshift-kni/commatrix/debug"
 	"github.com/openshift-kni/commatrix/nodes"
 	"github.com/openshift-kni/commatrix/types"
 )
 
 const (
-	processeNameFieldIdx  = 5
 	localAddrPortFieldIdx = 3
 	interval              = time.Millisecond * 500
 	duration              = time.Second * 5
 )
 
-var (
-	// TcpSSFilterFn is a function variable in Go that filters entries from the 'ss' command output.
-	// It takes an entry from the 'ss' command output and returns true if the entry represents a TCP port in the listening state.
-	tcpSSFilterFn = func(s string) bool {
-		return strings.Contains(s, "127.0.0") || !strings.Contains(s, "LISTEN")
-	}
-	// UdpSSFilterFn is a function variable in Go that filters entries from the 'ss' command output.
-	// It takes an entry from the 'ss' command output and returns true if the entry represents a UDP port in the listening state.
-	udpSSFilterFn = func(s string) bool {
-		return strings.Contains(s, "127.0.0") || !strings.Contains(s, "ESTAB")
-	}
-)
-
-func CreateComDetailsFromNode(cs *client.ClientSet, node *corev1.Node, tcpFile, udpFile *os.File) ([]types.ComDetails, error) {
-	debugPod, err := debug.New(cs, node.Name, consts.DefaultDebugNamespace, consts.DefaultDebugPodImage)
+func CreateComDetailsFromNode(debugPod *debug.DebugPod, node *corev1.Node, tcpFile, udpFile *os.File) ([]types.ComDetails, error) {
+	ssOutTCP, err := debugPod.ExecWithRetry("ss -anpltH", interval, duration)
 	if err != nil {
 		return nil, err
 	}
-	defer func() {
-		err := debugPod.Clean()
-		if err != nil {
-			fmt.Printf("failed cleaning debug pod %s: %v", debugPod, err)
-		}
-	}()
-
-	ssOutTCP, err := debugPod.ExecWithRetry("ss -anplt", interval, duration)
+	ssOutUDP, err := debugPod.ExecWithRetry("ss -anpluH", interval, duration)
 	if err != nil {
 		return nil, err
 	}
-	ssOutUDP, err := debugPod.ExecWithRetry("ss -anplu", interval, duration)
-	if err != nil {
-		return nil, err
-	}
-
-	ssOutFilteredTCP := filterStrings(tcpSSFilterFn, splitByLines(ssOutTCP))
-	ssOutFilteredUDP := filterStrings(udpSSFilterFn, splitByLines(ssOutUDP))
 
-	_, err = tcpFile.Write([]byte(fmt.Sprintf("node: %s\n%s", node.Name, strings.Join(ssOutFilteredTCP, "\n"))))
+	_, err = tcpFile.Write([]byte(fmt.Sprintf("node: %s\n%s", node.Name, string(ssOutTCP))))
```

Comment:
Can you add an extra new line here? To separate between each node section

Reply:
done.